docs/developinstallwin.rst

```diff
@@ -0,0 +1,147 @@
+   to do this everytime updates from the repositories are pulled.
+
+   **Before** compilation in Windows, make sure to delete ``AvaFrame/build`` directory, in addition to any .pyd, .c, and
+   .pycache files in ``AvaFrame/avaframe/com1DFA``
+
+Install avaframe and its requirements by **either** doing::
+
+  pip install -e .
+
+or if this fails (see `github issue 986 <https://github.com/avaframe/AvaFrame/issues/986>`_), do::
+
+  python setup.py develop
+
+This installs avaframe in editable mode, so every time you import avaframe the
+current (local) version will be used.
+
+Test it by starting ``python`` and do an ``import avaframe``. If no error comes
+up, you are good to go.
+
+If wanted: Integrate conda environment into editor of your choice (i.e. PyCharm, Spyder or any other)
+
+Head over to :ref:`gettingstarted:First run` for the next steps.
+
+Using QGIS from Conda
+^^^^^^^^^^^^^^
+It might be wanted to both have a script installation and the plugin installed on your machine at the same time.
+Unfortunately, installing AvaFrame on Windows via QGIS plugin might overrule your script AvaFrame installation, causing
+issues when trying to work on both versions in parallel. This depends on your mode of installation (direct installer, conda, OSGeo4W...).
+
+Below, we have compiled a **short guide**, describing a way to install QGIS and the Connector plugin in a separate
+*conda* environment. This method will result in two script installations (if the steps above have been followed),
+where one will be used by QGIS and the other may be worked on separately, preventing any overlap.
+
+Firstly, make sure that if you have another QGIS installation somewhere, that the AvaFrame plugin is not installed.
+Next, create and activate a new environment, which will contain your QGIS installation. The name of the environment
+can be anything, but since we will install the latest version of QGIS, we will call it *qgis_latest* in this example.
+Open your conda terminal and use::
+
+  conda create -n qgis_latest
+  conda activate qgis_latest
+
+To **install the latest QGIS version**, use::
+
+  conda install qgis --channel conda-forge
+
+.. Note::
+  Because conda will always try to install the latest version, if you want to use another version, you need to specify it, e.g.:
+
+  ``conda install qgis=3.34.11 --channel conda-forge``
+
+Now install avaframe into the qgis environment. This is to make sure that the plugin will find this version first, and install the necessary dependencies.
+In the conda terminal, use::
+
+  pip install avaframe
+
+Now find the directory where your environment is located. The path should be something like:
+``C:\Users\USER\miniconda3\envs\qgis_latest``. Change into it::
+
+  cd \miniconda3\envs\qgis_latest
+
+.. Note::
+  **Tip:** To find the path, you can do ``pip show avaframe`` and copy the
+  first part which shows your environment directory
```

Comment:
I think we could leave this out and just say: use a directory that's different then your script installation from above, right? Since then we would not touch any conda env directories. The requirements should come through the commands below anyways...

Reply:
I am not sure about this one.. I believe it's important to install the requirements before in the environment right? At least we need pip numpy cython. Or will the cloned AF be able to use dependencies outside the environment?

Reply:
That is a good point, installing avaframe however is not a good idea. Please switch `pip install avaframe` to `conda install pip numpy cython`. The rest of the requirements will be met via the pip install lower down.